Vizzuality/heco-invest
# feat: Location model and importer (with Colombia Wiki data)

First iteration of importing Colombia location data. I have fetched data from wiki where department names and list of municipalities can be found. 

Crawler also downloaded coordinates for every municipality from wiki --> "almost" all locations had coordinates there ;). I plan to use these municipality coordinates during next iteration when I add regions -- need to compute intersections between regions and municipalities. Because of complexity of these two tasks I have decided to split this to two separate PRs - one for administrative regions (which are covered by this PR) and second for geographic regions.

## Testing instructions

Just run `rake db:seed` and check that locations data were correctly imported ;)

## Tracking

https://vizzuality.atlassian.net/browse/LET-130
